package/remarkable-splash/package

```diff
@@ -10,8 +10,8 @@ timestamp=2019-12-31T10:07Z
 section="utils"
 maintainer="Eeems <eeems@eeems.email>"
 license=MIT
-installdepends=(display)
 flags=(patch_rm2fb)
+installdepends=(display)
```

Comment:
Looks like this change is a no-op.

Reply:
Yeah it was leftover from when I used to have the package part of this one. I can revert it fully.